docs/walkthrough.rst

```diff
@@ -0,0 +1,199 @@
+* Delete Key-Value pairs
+* Convert the namespaces of Key-Value pairs
+
+Following this walkthrough, we hope to give you an understanding of \
+the four scripts that will help you to manipulate your annotations.
+
+Object selection with the scripts
+---------------------------------
+
+The object selection logic is the same for all four scripts. In OMERO there are \
+two distinct hierarchies:
+
+* **Projects** → **Datasets** → **Images**
+* **Screens** → **Plates** → (**Wells** or **Acquisition/Run**) → **Images**
+
+More details about selecting High-Content-Screening are given in another \
+section (see :doc:`Selecting High-Content-Screening </extended_description>`).
+
+To select which objects to manipulate by batch, the scripts offer four distinct ways.
+
+Direct selection
+^^^^^^^^^^^^^^^^
+The first is to simply select the all the desired objects. Opening the script \
+after selecting the object will prefill the script parameters with the right \
+object type and IDs:
+
+.. figure:: images/annotation_01_selecting_all_autofill.png
+
+   *Selection of multiple datasets and the auto-filled Export to CSV script.*
+..
+
+For the direct selection, leave the Target Data Type as "*<selected>*" \
+(in this example, choosing **Dataset** would also work).
+
+Children selection
+^^^^^^^^^^^^^^^^^^
+Instead of selecting the objects one by one, we can select the parent object \
+and set the "*Target Data Type*" to the type of the children objects \
+we want to select.
+
+.. figure:: images/annotation_02_selecting_parentchild_autofill.png
+
+   *Selection of all the images of a project and the auto-filled Export to CSV script.*
+..
+
+In this example, we would select all the **Images** found under the **Project:701**.
+
+Tag selection
+^^^^^^^^^^^^^
+The third option is to select from a tag all the objects of a given type \
+(do not confiuse with the option of exporting tags). This grants additional  \
+flexibility, either to select objects attached to different projects \
+or different owners, or to have a finer control over \
+which objects to process.
+
+.. figure:: images/annotation_03_selecting_tags.png
+
+   *Selection of a subset of datasets from a tag and the auto-filled Export to CSV script.*
+..
+
+The selection works in the same way as for the children selection. Note \
+that choosing "*<selected>*" for "*Target Data Type*" will result in an \
+error in this case.
+
+All from selected
+^^^^^^^^^^^^^^^^^
+The fourth option is to apply the scripts to all of the selected objects and the children  \
+objects. Selecting a Project and using the option **<all (from selected)>** with result in \
+applying the scripts to the selected project, to the linked datasets and the linked images \
+bellow. Note that this option does not exist for the import of annotation from csv.
+
+Exporting annotations
+---------------------
+
+Exporting annotations will generate a .csv file with the header names \
+expected by the import .csv script. This has two use:
```

Comment:
```suggestion
expected by the import .csv script. This has two use:
```

Sorry, cannot suggest something reasonable. Please explain what ``This`` is in your sentence. ``...has two use..`` is certainly not correct English.

Reply:
Changed to:
`Exporting annotations creates a .csv file with headers compatible with the import script. It can be used for:`